feature/qos/ate_tests/qos_basic_test/qos_basic_test.go

```diff
@@ -1517,10 +1518,8 @@ func ConfigureJuniperQos(t *testing.T, dut *ondatra.DUTDevice) {
 	for _, tc := range classifierIntfs {
 		i := q.GetOrCreateInterface(tc.intf)
 		i.SetInterfaceId(tc.intf)
-		if deviations.ExplicitInterfaceRefDefinition(dut) {
-			i.GetOrCreateInterfaceRef().Interface = ygot.String(tc.intf)
-			i.GetOrCreateInterfaceRef().Subinterface = ygot.Uint32(0)
-		}
+		i.GetOrCreateInterfaceRef().Interface = ygot.String(tc.intf)
```

Comment:
what this test is not consistent with other tests in using deviations InterfaceRefConfigUnsupported(dut)  and deviations.IntfRefConfigUnsupported

Reply:
This is under ConfigureJuniperQos, it is specific to Juniper and the mentioned deviation is not applicable.